packages/bot-web-ui/src/components/dashboard/dbot-tours/mobile-tours/__tests__/onboarding-tour.spec.tsx

```diff
@@ -8,6 +8,7 @@ import { mock_ws } from 'Utils/mock';
 import RootStore from 'Stores/root-store';
 import { DBotStoreProvider, mockDBotStore } from 'Stores/useDBotStore';
 import OnboardingTour from '../onboarding-tour';
+import '@testing-library/react/dont-cleanup-after-each';
```

Comment:
This line needs to go above the importing of @testing-library/react (ex. `import { render, screen, waitFor } from '@testing-library/react'`). The test case will fail if the importing sequence breaks :pray: